src/app/plugins/connectfour.plugin.ts

```diff
@@ -129,6 +162,19 @@ class ConnectFourGame {
     return this.gameOver;
   }
 
+  public getWinner() {
+    return this._winner;
+  }
+
+  public getLoser() {
+    // -1 means playerA won
+    return this.getWinner() === -1 ? this._playerB : this._playerA;
```

Comment:
can we avoid magic numbers?

Reply:
We need it in one place, just inside of getWinner, but we can make `getWinner` return a user so that its not a magic number anymore